packages/vkui/src/components/HorizontalCell/HorizontalCell.tsx

```diff
@@ -25,6 +25,11 @@ const stylesSize = {
   auto: styles.sizeAuto,
 };
 
+const textAlignClassNames = {
+  center: styles.textAlignCenter,
+  end: styles.textAlignEnd,
+};
```

Comment:
Вроде как обычно же для значения по умолчанию создают поле со значением undefined? 
Тогда можно будет убрать проверку  `textAlign !== 'start'`
```suggestion
const textAlignClassNames = {
  start: undefined,
  center: styles.textAlignCenter,
  end: styles.textAlignEnd,
};
```

Reply:
Кажется, что лишних значений в мапе лучше не создавать, проще проверку бахнуть
По кодовой базе чаще именно проверки встречаются х)